src/main/resources/db/changelog/changesets/changelog_20240705T093039Z.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="souissimai (generated)" id="1720171859464-1">
+        <dropColumn columnName="STASHED" tableName="ELEMENT"/>
+    </changeSet>
+    <changeSet author="souissimai (generated)" id="1720171859464-2">
+        <dropColumn columnName="STASH_DATE" tableName="ELEMENT"/>
+    </changeSet>
```

Comment:
We should manage only one changeset for both dropColumn actions here I think.

Reply:
done